pkgs/tools/filesystems/bcachefs-tools/update.sh

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env nix-shell
+#!nix-shell -i bash -p nix-prefetch-github git jq
+
+set -x -eu -o pipefail
+
+cd $(dirname "${BASH_SOURCE[0]}")
+
+rm -rf bcachefs-tools.git
+git clone --depth=1 --bare "http://github.com/koverstreet/bcachefs-tools.git"
+DATE=$(git --git-dir=bcachefs-tools.git log --date=short --pretty=format:%ad HEAD)
+rm -rf bcachefs-tools.git
+
+nix-prefetch-github koverstreet bcachefs-tools | jq --arg date $DATE '. + {date:$date}' > version.json
```

Comment:
I might be biased, but you could achieve the same using `nix-update --version=branch` without using json and `updateScript = nix-update-script { extraArgs = [ "--version=branch" ]; };`. Might be less code (also you might have to pull from github). But now the code written anyway...

Reply:
@Mic92 offtopic but do we have documentation on how to use `nix-update-script` in nixpkgs?